notebooks/link-analysis/visit_data.py

```diff
@@ -227,7 +227,7 @@ def top_next_URLs(current_URL, population_fraction=0.01):
 np.mean(test_data['top_10'])
```

Comment:
I think this belongs in a separate branch where you did the link-analysis work. Will revisit this later and compare if we already have this.